packages/legend-application/src/components/VirtualAssistant.tsx

```diff
@@ -379,7 +408,23 @@ const VirtualAssistantSearchPanel = observer(() => {
         <PanelLoadingIndicator
           isLoading={assistantService.searchState.isInProgress}
         />
-        {Boolean(results.length) && (
+        {assistantService.isSearchConfigOpen && (
```

Comment:
I'll think more about this approach. I'm rather iffy about this. Let's leave this comment here so we can address later.